docs/update_and_migration/from_4.6/update_from_4.6.md

```diff
@@ -594,3 +594,7 @@ To use the [latest features](ibexa_dxp_v4.6.md) added to them, update them separ
     ```bash
     composer require ibexa/product-catalog-date-time-attribute:[[= latest_tag_4_6 =]]
     ```
+
+## v4.6.23
+
+No additional steps needed.
```

Comment:
Just a placeholder for now.

TODO: Check if there are steps

Reply:
Nothing new in the `ibexa/installer` https://github.com/ibexa/installer/releases/tag/v4.6.23